hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -1596,6 +1596,7 @@
                                 </xs:sequence>
                             </xs:complexType>
                         </xs:element>
+                        <xs:element name="acknowledge-type" type="wan-ack-type" minOccurs="0" maxOccurs="1" default="ACK_ON_OPERATION_COMPLETE"/>
```

Comment:
I remember we discussed that previously. `default` seems unused but people can get confused with the value. @bilalyasar ?